Packs/PAN-OS/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "PAN-OS",
+    "description": "PAN-OS content pack.",
+    "support": "xsoar",
+    "currentVersion": "1.0.0",
+    "author": "Cortex XSOAR",
+    "url": "https://www.paloaltonetworks.com/cortex",
+    "email": "",
+    "categories": [
+        "Utilities", "Network Security"
+    ],
+    "tags": [
+        "PAN-OS"
```

Comment:
```suggestion
        "PAN-OS",
        "NetOps",
        "Panorama"
```

Reply:
Not sure if it should go to here or to key words.
but we definitely want to add use cases as well to this pack.